### PR TITLE
Template documentation

### PR DESCRIPTION
**Pull request by veulkehc**

### CODE REVIEW

File: source/user_manual.rst [hunk 1]
```diff
@@ -556,6 +558,37 @@ Mailing Lists work independently from Routes.
 If there is a Mailing List or Route with the same address, the incoming message will hit the Route and Mailing List simultaneously. This can be pretty convenient for processing replies to the Mailing List and integrating into things like forums
 or commenting systems.
 
+.. _templating:
+
+Templating
+==========
+
+Mailgun allows to store predefined templates via :ref:`Template API <api-templates>` and use them to send messages via :ref:`sending api <api-sending-messages>` just providing template id.
```
Comment (highlycaffeinated) [inline source/user_manual.rst]: "Mailgun allows you to store"

File: source/api-sending.rst [hunk 2]
```diff
@@ -49,6 +49,9 @@ This makes sense for parameters like ``cc``, ``to`` or ``attachment``.
  inline                Attachment with ``inline`` disposition. Can be used to
                        send inline images (see :ref:`example <inline-image>`).
                        You can post multiple ``inline`` values.
+ template              ID of a template stored via :ref:`template API <api-templates>`.
+                       Please not ``html`` parameter is ignored if you are sending messages with this parameter.
```
[the remark this answers is not in the document]
Comment (kalebpomeroy) [reply source/api-sending.rst]: html and text parameters are both ignored, right?

[the remark this answers is not in the document]
Comment (veulkehc) [reply source/api-sending.rst]: @kalebpomeroy Only html is ignored as template replace html part. Text part is allowed.

**Comment by veulkehc** [comment]
@highlycaffeinated 

- Yes, this is intentional. I noticed broken links and decided to fix them. They are minor.
- https://github.com/mailgun/mailgun-go has to be updated.
- Will fix that.
- Will do

**Pull request merged**